apps/dh/api-dh/project.json

```diff
@@ -70,7 +70,7 @@
         "commands": ["npx ts-node tools/dh/api-dh/ci/stop-api-dh.ts"],
         "parallel": false
       },
-      "outputs": ["apps/dh/api-dh/source/DataHub.WebApi/bin/Release/net6.0"]
```

Comment:
Question: are we sure this is right?

Reply:
Resolving this comment since it's similar to the other one.